src/Plugin/CKEditorPlugin/DrupalEntity.php

```diff
@@ -25,6 +26,17 @@ class DrupalEntity extends EmbedCKEditorPluginBase {
   /**
    * {@inheritdoc}
    */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    $configuration['inline'] = $container->get('config.factory')
```

Comment:
I'm not sure it's going to be possible to make it configurable per button. The reason being that making drupal-entity elements inline requires changing the DTD of the editor instance, which will take effect for the entire editor, regardless of how many different embed buttons are loaded into that instance. It's intrinsically a per-editor thing, not a per-button thing. But maybe I'm misunderstanding you -- what did you have in mind?